apps/src/applab/designMode.js

```diff
@@ -190,6 +192,18 @@ designMode.fontFamilyOptionFromStyle = function(style) {
   return applabConstants.fontFamilyOptions[fontIndex === -1 ? 0 : fontIndex];
 };
 
+designMode.assignImageType = function(element, image_source) {
+  if (ICON_PREFIX_REGEX.test(image_source)) {
+    return 'icon';
```

Comment:
[[same as this comment](https://github.com/code-dot-org/code-dot-org/pull/36637#discussion_r487330002)] we should move these image type values into a constant/type as well